src/NCalc/Expression.cs

```diff
@@ -193,9 +193,6 @@ public bool HasErrors()
 
         public LogicalExpression ParsedExpression { get; private set; }
 
-        protected Dictionary<string, IEnumerator> ParameterEnumerators;

```

Comment:
Agreed, can we increase a minor version in release build?

Reply:
If you'd like to avoid breaking changes, we can leave it as a global. But `ParametersBackup` variable is protected too...

Reply:
Sounds reasonable.

https://github.com/sklose/NCalc2/blob/cd8d88ea1738893919c3e4db6d01bf3b0d8193ef/src/NCalc/NCalc.csproj#L6

Reply:
> Sounds reasonable.
> 
> https://github.com/sklose/NCalc2/blob/cd8d88ea1738893919c3e4db6d01bf3b0d8193ef/src/NCalc/NCalc.csproj#L6

Do I understand you correctly, just increasing the version in csproj is required? The AppVeyor still shows 3.0.216 thought https://ci.appveyor.com/project/sklose/ncalc2/builds/49477816

Reply:
> maybe we will mark these fields with Obsolete attribute in a separate PR,

I think you can just get rid of them, but I'm okay with the Obsolete suggestion as well.  Why not do it on this PR though?  

> Do I understand you correctly, just increasing the version in csproj is required?

Sorry, my bad.  I pointed you in the wrong direction.  I've only done this once before, but I think this needs to be modified since the nuget package gets the appbeyor build #.
https://github.com/sklose/NCalc2/blob/cd8d88ea1738893919c3e4db6d01bf3b0d8193ef/appveyor.yml#L1

Reply:
> Why not do it on this PR though?

I don't understand how the process is organized. It's seems every PR is squashed and a new version is automatically released. So, a separate PR is needed?

Reply:
> Sorry, my bad. I pointed you in the wrong direction. I've only done this once before, but I think this needs to be modified since the nuget package gets the appbeyor build #.
> 
> https://github.com/sklose/NCalc2/blob/cd8d88ea1738893919c3e4db6d01bf3b0d8193ef/appveyor.yml#L1

Now seems to be ok, do I need to rollback the csproj changes?

Reply:
I'm not sure there's much of a process. ` [Obsolete]` is a warning by default, not an error so it wouldn't require a new major version.  It seems reasonable to just add it on this PR rather than creating a new one that has to be reviewed.

Reply:
So, we decided not to introduce a breaking change and just mark them obsolete for now?

Reply:
I was leaving it up to you since its your PR.  

> I think you can just get rid of them (properties), but I'm okay with the Obsolete suggestion as well